package.json

```diff
@@ -10,7 +10,7 @@
     "@fortawesome/react-fontawesome": "^0.2.0",
     "@headlessui/react": "^1.7.18",
     "@headlessui/tailwindcss": "^0.2.0",
-    "@kittycad/lib": "^0.1.0",
+    "@kittycad/lib": "^0.0.56",
```

Comment:
not really sure what happened here, I just bump the version in the library's package.json as it was 0.55 but looks like there was a release where the version didn't match the package.json's version 🤷

Reply:
This was me sorry